src/main/java/io/confluent/connect/jdbc/JdbcSourceConnectorConfig.java

```diff
@@ -155,14 +155,16 @@
   public static final String TABLE_TYPE_CONFIG = "table.types";
   private static final String TABLE_TYPE_DOC =
       "By default, the JDBC connector will only detect tables with type TABLE from the source Database. "
-      + "This config allows a command seperated list of table types to extract. Optons include:\n "
-      + "* TABLE "
-      + "* SYSTEM TABLE "
-      + "GLOBAL TEMPORARY "
-      + "* LOCAL TEMPORARY "
-      + "* ALIAS "
-      + "* SYNONYM "
-      + "In most cased it only makes sense to have either Table or View, depending on the usecase. ";
+      + "This config allows a command seperated list of table types to extract. Optons include:\n"
+      + "* TABLE\n"
+      + "* VIEW\n"
+      + "* SYSTEM TABLE\n"
+      + "* GLOBAL TEMPORARY\n"
+      + "* LOCAL TEMPORARY\n"
+      + "* ALIAS\n"
+      + "* SYNONYM\n"
+      + "In most cased it only makes sense to have either Table or View, depending on the usecase.";
```

Comment:
Typo: "cased". Also, maybe just leave off "depending on the usecase"? Seems redundant.